grass7/temporal/t.rast.what.aggr/t.rast.what.aggr.py

```diff
@@ -34,14 +34,22 @@
 
 #%option G_OPT_DB_COLUMN
 #% key: date_column
-#% description: Name of the input column containing dates for aggregates
+#% description: Name of the starting column containing dates for aggregates
 #% required: no
 #%end
 
 #%option
 #% key: date
 #% type: string
-#% description: The date for aggregates
+#% description: The starting date for aggregation
+#% required: no
+#% multiple: no
+#%end
+
+#%option
+#% key: final_date
+#% type: string
+#% description: The end date for aggregation, require date option
```

Comment:
```suggestion
#% description: The end date for aggregation, requires date option
```

Reply:
I'd suggest to use start_date and end_date as option names; it's consistent with other modules and less confusing